newsroom/settings.py

```diff
@@ -77,8 +77,7 @@ def get_client_config():
     for key, setting in (get_setting() or {}).items():
         if key not in ['_updated', 'version_creator']:
             value = setting.get('value', setting.get('default'))
-            if value:
```

Comment:
Also @petrjasek Is it safe here to remove this check. It was necessary for assigning null value for `newsonly` filter. It didn't caused any issues locally though.